python/lsst/ts/m2gui/controltab/tab_cell_status.py

```diff
@@ -63,6 +63,9 @@ def __init__(self, title, model):
 
         self._actuator_data_selection = self._create_actuator_data_selection()
 
+        # Timer to update the forces on cell status
```

Comment:
# Time to update cell status forces (displays?)

Reply:
Updated in the commit: [24c98f6](https://github.com/lsst-ts/ts_m2gui/pull/22/commits/24c98f6e555c1f5d264f9e6b7db25ba443faf54b). Thanks!